docs/deploy-cephfs.md

```diff
@@ -7,26 +7,28 @@ The CSI CephFS plugin is able to both provision new CephFS volumes and attach an
 CSI CephFS plugin can be compiled in a form of a binary file or in a form of a Docker image. When compiled as a binary file, the result is stored in `_output/` directory with the name `cephfsplugin`. When compiled as an image, it's stored in the local Docker image store.
 
 Building binary:
+
 ```bash
-$ make cephfsplugin
+make cephfsplugin
 ```
 
 Building Docker image:
+
 ```bash
-$ make image-cephfsplugin
+make image-cephfsplugin
 ```
 
 ## Configuration
 
 **Available command line arguments:**
 
-Option | Default value | Description
------- | ------------- | -----------
-`--endpoint` | `unix://tmp/csi.sock` | CSI endpoint, must be a UNIX socket
-`--drivername` | `csi-cephfsplugin` | name of the driver (Kubernetes: `provisioner` field in StorageClass must correspond to this value)
-`--nodeid` | _empty_ | This node's ID
-`--volumemounter` | _empty_ | default volume mounter. Available options are `kernel` and `fuse`. This is the mount method used if volume parameters don't specify otherwise. If left unspecified, the driver will first probe for `ceph-fuse` in system's path and will choose Ceph kernel client if probing failed.
-`--metadatastorage` | _empty_ | Whether should metadata be kept on node as file or in a k8s configmap (`node` or `k8s_configmap`)
+Option              | Default value         | Description
+--------------------|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+`--endpoint`        | `unix://tmp/csi.sock` | CSI endpoint, must be a UNIX socket
+`--drivername`      | `csi-cephfsplugin`    | name of the driver (Kubernetes: `provisioner` field in StorageClass must correspond to this value)
+`--nodeid`          | _empty_               | This node's ID
+`--volumemounter`   | _empty_               | default volume mounter. Available options are `kernel` and `fuse`. This is the mount method used if volume parameters don't specify otherwise. If left unspecified, the driver will first probe for `ceph-fuse` in system's path and will choose Ceph kernel client if probing failed.
```

Comment:
this makes the option wrapped in two lines

Reply:
@rootfs https://github.com/ceph/ceph-csi/blob/2888640157c37226266b323f96eecfb97c680e5e/docs/deploy-cephfs.md I dont see it, may be I am missing something